setup.py

```diff
@@ -113,6 +113,7 @@ def get_flash_attention_extensions(cuda_version: int, extra_compile_args):
                 "nvcc": extra_compile_args.get("nvcc", [])
                 + [
                     "-O3",
+                    "-std=c++17",
```

Comment:
Can this be conditionned on Windows builds only?

Reply:
https://github.com/facebookresearch/xformers/pull/460/commits/bf5b1fd922bec9a7723723e1d014e7d09a60fcea should do it